content/using/install.md

```diff
@@ -232,6 +232,8 @@ Or, if you'd prefer to copy your key in, you can run:
 
 Either command will create a directory called `sampel-palnet/` and begin building your ship. It may take a few minutes.
 
+Dojo might be unresponsive for some time (minutes) during first boot, for example you might see a `<<http>>` or `<<behn>>` spinner and keyboard is unresponsive.
```

Comment:
But what is it doing? If you add a short "an indication of activity in eyre, urbit's http vane" and something like that for behn that would be great.

Reply:
Bumping this pr.

Reply:
Apologies, have way too little knowledge yet to explain what is happening, so will close.